### ISSUE TITLE
tabs: with "md-center-tabs" has layout issues

### ISSUE THREAD

**Issue by RostYanevych** [angular/material#11566]
## Bug, enhancement request, or proposal:
`md-tabs` with `md-center-tabs="true"` does not show tabs at all. It's easy to reproduce on `md-tabs` demo by just adding `md-center-tabs="true"` to `<md-tabs>` tag. 
# CodePen and steps to reproduce the issue:
 https://codepen.io/anon/pen/vvmQPg

#### Detailed Reproduction Steps: 
1. Use the markup 
```html
<md-tabs md-dynamic-height="" md-border-bottom="" md-center-tabs="true">...
```

### What is the expected behavior?
tabs are shown centered

### What is the current behavior?
tabs are not shown at all. In some cases they are shown on the right side of container

### Which versions of AngularJS, Material, OS, and browsers are affected?

- AngularJS: 1.7.5
- AngularJS Material: 1.1.11
- OS: Win 7 x64
- Browsers: Firefox, IE-11
 
### Is there anything else we should know? Stack Traces, Screenshots, etc.
It's working well in angular-material version 1.1.10

**Comment by Splaktar** [comment]
I can reproduce this in Chrome as well. I made some minor tweaks to the [CodePen](https://codepen.io/Splaktar/pen/zydGro). If you click on the tabs header, the tabs appear, but that should not be required.

On Chrome and Safari, when the tabs display, they are in the center. But on Firefox the tabs are on the right side and some are not in the viewport:
![screen shot 2018-12-26 at 2 35 13 pm](https://user-images.githubusercontent.com/3506071/50455375-96c87f80-091b-11e9-8872-5bce23e653e0.png)

**Comment by Splaktar** [comment]
This was caused by the last iteration of [the fix for IE11 pagination](https://github.com/angular/material/pull/11432) where tabs always showed up as truncated (I.e. "Ta..."). This fix was to resolve a regression caused by `v1.1.2`.
